client/assets/stylesheets/_nav-unification.scss

```diff
@@ -11,38 +11,29 @@
  */
 
 .is-nav-unification {
+	// breakpoint used in wp-admin
+	@media only screen and ( min-width: 782px ) {
+		--masterbar-height: 32px;
```

Comment:
I think you should note in a comment that this overides  the original which is set in `client/assets/stylesheets/shared/_variables.scss`

Reply:
Added https://github.com/Automattic/wp-calypso/pull/46035/commits/bb7a1438c65165f95a5cc0342ec7f77d7ed01ca7